src/java.base/share/classes/javax/crypto/ProviderVerifier.java

```diff
@@ -51,7 +51,7 @@ final class ProviderVerifier {
     private CryptoPermissions appPerms = null;
 
     /**
-     * Creates a ProviderVerifier object to verify the given URL.
+     * Creates a <code>ProviderVerifier</code> object to verify the given URL.
```

Comment:
Use `{@code }` for all of the javadoc in this class?

Reply:
using `{@code }`